cle/backends/elf/symbol.py

```diff
@@ -42,6 +42,7 @@ def __init__(self, owner, symb):
 
         self.elftype = realtype
         self.binding = symb.entry.st_info.bind
+        self.is_hidden = symb.entry['st_other']['visibility'] == 'STV_HIDDEN'
```

Comment:
I feel like rather than an attribute, we should cache the visibility type and use a `symbol.visibility` property that returns the corresponding integer value, which can then be compared against the module-global definitions for each one, like we do with `symboled.TYPE_OBJECT` et al.

Reply:
I like this proposal. @subwire what do you think?

Reply:
Just FYI, once #169 is approved, I've already got an `Enum` for this too. It doesn't seem to actually be used anywhere yet so it's a simple fix.